package.json

```diff
@@ -3,8 +3,8 @@
   "version": "2.5.1",
   "description": "React components for Stripe.js and Stripe Elements",
   "main": "dist/react-stripe.js",
-  "module": "dist/react-stripe.esm.js",
```

Comment:
I ran `npx publint` and it gave the suggestion to rename the exports here from `js` to `mjs`. 

When consuming this package there might be some ambiguity onto wether this file is ESM or Common JS (because it ends with .js). Changing the file extension to `.mjs` explicitly calls out that this files uses ESM